Fix applications count in workshop card

VitaliyYuras opened the pull request.

VitaliyYuras committed:
Fix applications count in workshop card

VadymLevkovskyi commented on OutOfSchool/OutOfSchool.DataAccess/Repository/ApplicationRepository.cs:
```diff
@@ -83,4 +84,17 @@ public async Task<int> UpdateAllApprovedApplications()
+    {
+        List<Application> applicationsToDelete = await dbContext.Applications
+        .Where(app => app.ChildId == childId)
+        .ToListAsync();
```
(Code style) Please fix code indentation.

VadymLevkovskyi reviewed (commented):
I see two major changes in code - adding deletion of child applications and changed `WorkshopProviderViewCard.AmountOfPendingApplications` mapping profile. 
Could they both be tested? Could other uncovered lines be tested as well? 

Also, please see minor code formatting issue in separate comment.

VitaliyYuras committed:
Added tests

VitaliyYuras commented:
> I see two major changes in code - adding deletion of child applications and changed `WorkshopProviderViewCard.AmountOfPendingApplications` mapping profile. Could they both be tested? Could other uncovered lines be tested as well?
> 
> Also, please see minor code formatting issue in separate comment.

Fixed
